src/main/java/com/sonyericsson/jenkins/plugins/bfa/statistics/Statistics.java

```diff
@@ -198,7 +198,7 @@ public Statistics(@JsonProperty("projectName")    String projectName,
     }
 
         /**
-         * @deprecated, kept for backwards compatibility.
+         * deprecated, kept for backwards compatibility.
```

Comment:
Why remove `@deprecated`?

Reply:
JAVADOC complained that @deprecated is not correct and suggested to switch it. I am not set on this change and can just ignore that JAVADOC error in my IDE

Reply:
The switching is probably because it should be at the bottom of the javadoc and will then be missing a title.
So please fix as the description of why the deprecation is will be lost otherwise.